### DISCUSSION TITLE
Little activity only recently, but nothing has moved in 2 years. Is this actively supported?

### DISCUSSION THREAD

**Discussion by prodkt** [roxiness/routify#521] [General]
I believe the answer is no, this is no longer supported nor is v3 actively being developed given its been 2 years since movement has been made on this. Looking to confirm so we don't leave this open as an option for upcoming projects.

**Comment by jakobrosenberg** [comment]
Hi @prodkt . Thank you for your interest and sorry about the slow reply.

Routify 3 is actively being developed.

To put some numbers on it, there were 3 releases last month and 5 in the month before. Currently most efforts are aimed at documentation, templates/examples and polishing. 🙂
